mitmproxy/protocol/http2.py

```diff
@@ -96,15 +96,16 @@ def __init__(self, ctx, mode):
         self.server_to_client_stream_ids = dict([(0, 0)])
         self.client_conn.h2 = SafeH2Connection(self.client_conn, client_side=False, header_encoding=False)
 
-        # make sure that we only pass actual SSL.Connection objects in here,
-        # because otherwise ssl_read_select fails!
-        self.active_conns = [self.client_conn.connection]
-
```

Comment:
After discussing this on Slack: We won't support this soon anyway due to the ALPN requirements (which would require a major rewrite), so the changes here are fine for now.